lib/tasks/solid_errors_tasks.rake

```diff
@@ -0,0 +1,18 @@
+namespace :solid_errors do
+  desc "Copy and run migrations from Solid Errors"
+  task install_migrations: :environment do
+    source = File.expand_path("../../solid_errors/db/migrate", __FILE__)
+    destination = File.join(Rails.root, "db", "migrate")
+
+    Dir.glob("#{source}/*.rb").each do |migration|
+      original_filename = File.basename(migration)
+      timestamp = Time.now.strftime('%Y%m%d%H%M%S_')
+      filename_with_timestamp = "#{timestamp}#{original_filename}"
```

Comment:
```suggestion
      timestamp = Time.now.strftime('%Y%m%d%H%M%S')
      filename_with_timestamp = "#{timestamp}_#{original_filename}"
```

Reply:
This does not seem correct. Rails migrations have an underscore after the timestamp but this suggestion removes it. Not sure why?

Reply:
Removed previous comment. I did not see the moved underscore.